js/src/framework.ts

```diff
@@ -512,7 +512,8 @@ export function reportEvaluatorResult(
     if (!verbose && !jsonl) {
       console.error(warning("Add --verbose to see full stack traces."));
     }
-  } else if (summary) {
+  }
+  if (summary) {
```

Comment:
This is now aligned with how we do it in the python framework, where even if there are errors in the eval, we still print out the summary.